File: src/Workflow.Domain/Forms/HR/TravelDetail.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Text;
using System.Threading.Tasks;

namespace Workflow.Domain.Entities.HR
{
    [DataContract]
    public class TravelDetail
    {
        [DataMember(Name = "id")]
        public int Id { get; set; }

        [DataMember(Name = "classTravelEntitlement")]
        public byte ClassTravelEntitlement { get; set; }

        [DataMember(Name = "classTravelRequest")]
        public byte ClassTravelRequest { get; set; }

        [DataMember(Name = "reasonException")]
        public string ReasonException { get; set; }

        [DataMember(Name = "purposeTravel")]
        public byte PurposeTravel { get; set; }

        [DataMember(Name = "estimatedCostTicket")]
        public decimal EstimatedCostTicket { get; set; }

        [DataMember(Name = "requestHeaderId")]
        public int RequestHeaderId { get; set; }

        [DataMember(Name = "costTicket")]
        public decimal costTicket { get; set; }

        [DataMember(Name = "reasonTravel")]
        public string ReasonTravel { get; set; }

        [DataMember(Name = "noRequestTaken")]
        public int NoRequestTaken { get; set; }

        [DataMember(Name = "extraCharge")]
        public decimal ExtraCharge { get; set; }

        [DataMember(Name = "remark")]
        public string Remark { get; set; }
    }
}

File: src/Workflow.Web.Service/Workflow/app/model/attForm/TravelDetail.js
Ext.define('Workflow.model.attForm.TravelDetail', {
	extend: 'Workflow.model.Base',
	
	fields: [
		{ name: 'id', type: 'int' },
        { name: 'classTravelEntitlement', type: 'int' },
        { name: 'classTravelRequest', type: 'int' },
        { name: 'reasonException', type: 'string' },
        { name: 'purposeTravel', type: 'string' },
        { name: 'estimatedCostTicket', type: 'float' },
        { name: 'requestHeaderId', type: 'int' },
        { name: 'costTicket', type: 'float' },
        { name: 'reasonTravel', type: 'string' },
        { name: 'noRequestTaken', type: 'int' },
        { name: 'extraCharge', type: 'float' },
        { name: 'remark', type: 'string' }
        
	],
	proxy: {
	    writer: { writeAllFields: true },
	    type: 'rest',
	    actionMethods: {
	        read: 'GET',
	        destroy: 'DELETE',
	        update: 'PUT',
	        create: 'POST'
	    },
	    reader: {
	        type: 'json',
	        rootProperty: 'Result',
	        totalProperty: 'TotalRecords'
	    }
	}
});
